bindings/rust/src/lib.rs

```diff
@@ -105,6 +107,33 @@ impl Drop for KZGSettings {
     }
 }
 
+impl Bytes48 {
+    pub fn from_bytes(bytes: &[u8]) -> Result<Self, Error> {
+        if bytes.len() != 48 {
+            return Err(Error::InvalidBytes48(format!(
+                "Invalid byte length. Expected {} got {}",
+                48,
+                bytes.len(),
+            )));
+        }
+        let mut new_bytes = [0; 48];
+        new_bytes.copy_from_slice(bytes);
+        Ok(Self { bytes: new_bytes })
+    }
+
+    pub fn to_proof(self) -> Result<KZGProof, Error> {
```

Comment:
Are we using these helper functions? If not, maybe we can remove them? If yes, i think they should do the `validate_kzg_G1` logic since we consider KZGProof and KZGCommitment to be trusted types.

Reply:
Good call. They weren't actually being used. Removed.